contracts/src/v0.7/KeeperBase.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.7.6;
```

Comment:
We don't want to enforce a version on interfaces, because then consumers must use that. Use this for public interface contracts:

```suggestion
pragma solidity ^0.7.0;
```

Reply:
done